qa/tasks/rgw_multisite_tests.py

```diff
@@ -79,40 +78,16 @@ def setup(self):
         tests.realm_meta_checkpoint(realm)
 
     def begin(self):
-        # extra arguments for nose can be passed as a string or list
+        # extra arguments can be passed as a string or list
         extra_args = self.config.get('args', [])
         if not isinstance(extra_args, list):
             extra_args = [extra_args]
-        argv = [__name__] + extra_args
 
         log.info("running rgw multisite tests on '%s' with args=%r",
                  self.module_path, extra_args)
 
-        # run nose tests in the module path
-        conf = nose.config.Config(stream=get_log_stream(), verbosity=2, workingDir=self.module_path)
-        assert nose.run(argv=argv, config=conf), 'rgw multisite test failures'
-
-
-def get_log_stream():
-    """ return a log stream for nose output """
-    # XXX: this is a workaround for IOErrors when nose writes to stderr,
-    # copied from vstart_runner.py
-    class LogStream(object):
-        def __init__(self):
-            self.buffer = ""
-
-        def write(self, data):
-            self.buffer += data
-            if "\n" in self.buffer:
-                lines = self.buffer.split("\n")
-                for line in lines[:-1]:
-                    log.info(line)
-                self.buffer = lines[-1]
-
-        def flush(self):
-            pass
-
-    return LogStream()
+        argv = [sys.executable, self.module_path + "../test_multi.py"] + extra_args
+        testsuite = subprocess.run(argv, check=True)
```

Comment:
i don't think this can work. `test_multi.py` is creating its own ceph clusters with the vstart.sh script, but this teuthology script is meant to run tests against the clusters that teuthology itself set up

in `qa/tasks/rgw_multisite.py`, which runs just before this `rgw_multisite_tests.py` task, we do a lot of work to set up the python environment according to teuthology's deployment, and we pass this information into the python module [here](https://github.com/ceph/ceph/blob/master/qa/tasks/rgw_multisite_tests.py#L78-L79)

so we need to run the test cases in that same environment - i don't think a subprocess can work here